<launch>
    <node pkg="autosail" type="sim_converter_node.py" name="sim_helper"/>
    <node pkg="autosail" type="boat_tf_broadcaster.py" name="boat_tf_broadcaster"/>
    <node pkg="autosail" type="route_planner_service.py" name="route_planner_service"/>
    <remap from="/imu/data" to="sensors/imu" />
    <remap from="/gps/fix" to="sensors/gps" />
    <remap from="/wind_sensor" to="/wind/apparent" />
    <!--<remap from="/gps/fix_velocity" to="sensors/gps_velocity" /> -->
    <!--<remap from="/sail_controller/sail_angle" to="/main_sail_position/command"/>-->
    <!--<remap from="/rudder_controller/rudder_angle" to="/rudder_position/command"/>-->
    <node pkg="autosail" type="true_wind_node.py" name="true_wind"/>
    <include file="$(find autosail)/launch/ctrl.launch"/>
    <node pkg="topic_tools" type="relay" name="sail_relay" args="/main_sail_position/command /fore_sail_position/command"/>
    <include file="$(find autosail)/launch/path_planner.launch"/>
</launch>